config/settings.yml

```diff
@@ -163,7 +163,7 @@ defaults: &defaults
   ####################
   mongodb:
     provision:
-      plan_id: bc158c9a-7934-401e-94ab-057082a5073f
+      #plan_id: bc158c9a-7934-401e-94ab-057082a5073f
```

Comment:
Why is this commented?

Reply:
Unit tests check for mongodb not configured scenario as well. So that UT would fail with this being present.

Reply:
ok.